Documentation/botr/readytorun-format.md

```diff
@@ -326,6 +382,47 @@ This section marks region that contains thunks for ``READYTORUN_HELPER_DelayLoad
 helper. It is used by debugger for step-in into lazily resolved calls. It should not be required when 
 debuggers are able to handle debug info stored separately.
 
+## READYTORUN_SECTION_AVAILABLE_TYPES
+
+This section contains a native hashtable of all defined & export types within the compilation module. The key is a checksum of the full type name, the value is the exported type or defined type token row ID left-shifted by one and or-ed with bit 0 defining the token type:
+
+| Bit value | Token type |
+|----------:|:-----------|
+| 0         | defined type |
+| 1         | exported type |
+
+The checksum algorithm is implemented in [vm/versionresilienthashcode.cpp](https://github.com/dotnet/coreclr/blob/ec2a74e7649f1c0ecff32ce86724bf3ca80bfd46/src/vm/versionresilienthashcode.cpp#L75).
```

Comment:
Checksums are used to detect data errors. This is hashcode, not checksum.

Reply:
OK, so it's just about the terminology thing checksum -> hash? Thanks for pointing that out, I'll fix that.

Reply:
Corrected in 2nd commit.

Reply:
Right, https://en.wikipedia.org/wiki/Checksum